documentation/appendix_deploying_kubernetes_openshift_cluster.adoc

```diff
@@ -0,0 +1,38 @@
+[appendix]
+== Installing Kubernetes and OpenShift cluster
+
+The easiest way to get started with Kubernetes or OpenShift is using the `Minikube`, `Minishift` or `oc cluster up`
+utilities. This section provides a basic guidance on how to use them. More details are provided on the websites of
+the tools themselves.
+
+=== Kubernetes
+
+In order to interact with a Kubernetes cluster the https://kubernetes.io/docs/tasks/tools/install-kubectl/[`kubectl`]
+utility needs to be installed.
+
+The easiest way to get a running Kubernetes cluster is using `Minikube`. `Minikube` can be downloaded and installed
+from https://kubernetes.io/docs/getting-started-guides/minikube/[Kubernetes website]. Once installed, it can be started
+using:
+
+[source]
+minikube start
```

Comment:
Should we mention that it uses Virtualbox as default which is needed for that ? I use minikube with kvm so the --vm-driver option is needed.

Reply:
TBH, I think we should not have this in the docu at all ... we should not write the documentation for Minikube or Minishift because we will need to maintain it. I kept it only because it was already in the README.md, but I wouldn't go into even more details.

Reply:
I agree we shouldn't get sucked into providing more details, but I do think this would be helpful to people wanting to get started who aren't _au fait_ with K8s (or are _au fait_ with K8s, but not with OS). There's not so much documentation here that maintaining it is burdensome.

Reply:
@tombentley you mean we should keep the appendix as it is? Or we should add more details about virtualisation?